src/lib.rs

```diff
@@ -36,17 +36,17 @@
 //! Cloudevents uses a set of [feature flags] to conditionally compile
 //! only the module associated with the Protocol Binding you need:
 //!
-//! - `actix`: Enables the [actix] protocol binding module. This
+//! - `actix`: Enables the [`actix-web`](actix_web) protocol binding module. This
```

Comment:
I was thinking you'd link to `[binding::actix]` here. And then from there it has a link to `actix_web`.